sdk/test/Services/EC2Protocol/Generated/Model/Internal/MarshallTransformations/EndpointOperationRequestMarshaller.cs

```diff
@@ -59,6 +59,8 @@ public IRequest Marshall(EndpointOperationRequest publicRequest)
 
             if(publicRequest != null)
             {
+            
+            request.HostPrefix = $"foo.";
```

Comment:
This is using `request` but it is in a check for `publicRequest != null`. This doesn't look right and the indentation should be increased by 1 even in the case below where it doesn't have the if check.

Reply:
The indentation was wrong so I fixed it. But, it is correct to change the `request` object because that is what we are returning in the function. If you look a couple lines above as well, we alter the `Parameters` dictionary of the `request` object. We wouldn't want to alter the `publicRequest` object because that isn't returned to the user.